### PR TITLE
Misc improvements

### PR DESCRIPTION
**Pull request by dessaya**
- dashboard: fix overflow & chainblock
- wasp-cli: it is now possible to deposit L1 funds into a L2 ethereum account with: `wasp-cli chain deposit 0x34Cc8... iota:1000000`
- Fix several wasp-cli commands that were not working
- Fix most of cluster wasp-cli tests

### CODE REVIEW

File: tools/cluster/tests/wasp-cli_test.go [hunk 1]
```diff
@@ -87,14 +87,46 @@ func TestWaspCLI1Chain(t *testing.T) {
...
+
 func TestWaspCLIContract(t *testing.T) {
 	w := newWaspCLITest(t)
 
 	committee, quorum := w.CommitteeConfig()
 	w.Run("chain", "deploy", "--chain=chain1", committee, quorum)
 
 	// for running off-ledger requests
-	w.Run("chain", "deposit", "IOTA:10")
+	w.Run("chain", "deposit", "iota:10")
```
Comment (jorgemmsilva) [inline tools/cluster/tests/wasp-cli_test.go:129]: I know this is old code, but depositing 10 iotas shouldn't be possible :thinking: 
This is probably adjusted to the minimum storage deposit inside

**Review by jorgemmsilva — APPROVED** [review]

File: tools/cluster/tests/wasp-cli_test.go (hunk 1 above)
Comment (jorgemmsilva) [reply tools/cluster/tests/wasp-cli_test.go:129]: on a second thought, transfers from the cli are probably broken rn, @lmoe said he was going to work on that.

Comment (dessaya) [reply tools/cluster/tests/wasp-cli_test.go:129]: It is automatically adjusted, so in the end it deposits something like 100k.

Comment (jorgemmsilva) [reply tools/cluster/tests/wasp-cli_test.go:129]: yeah, I figured 👍 . This way the tests are weird to read, tho 😢

**Pull request merged**